redash/authentication/__init__.py

```diff
@@ -229,7 +223,7 @@ def redirect_to_login():
 def logout_and_redirect_to_index():
     logout_user()
 
-    if settings.MULTI_ORG and current_org == None:
+    if settings.MULTI_ORG and current_org is None:
```

Comment:
I don't know if this needs to be changed. Maybe we can keep `==`.

Reply:
OK, let's keep it as it is

Reply:
> OK, let's keep it as it is

OK.